openshift/4.0/templates/network-security-policy/internet-to-routes.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Template
```

Comment:
This simplifies ingress by allowing internet communication to pods that have published routes. No need to create one NSP per pod with route anymore.